doc/reference/views.rst

```diff
@@ -521,6 +521,19 @@ attributes:
     A field that can be aggregated.  This field will be used to compute the values
     for each cell.  If not set, the cohort view will count the number of occurrences.
 
+``<field>`` (optional)
+  allows to specify a particular field in order to manage it from the available measures, it's
+  main use is for hiding a field from the selectable measures:
+    * ``name`` (required)
+      the name of the field to use in the view.
+    * ``string`` (optional)
+      the name that would be used to display the field in the cohort view, overrides the
+      default python String attribute of the field.
+    * ``invisible`` (optional)
+      if true, the field will not appear either in the active measures nor in the selectable
+      measures (useful for fields that do not make sense aggregated, such as fields in different 
+      units, e.g. € and $).
+
```

Comment:
This doc can be included in the "IMP" commit I think.

Also, there are probably QUnit tests on pivot and graph view?
I think you will have to add some tests case for your new behavior to make sure it's included in the test suite.
Especially since it's a "core added feature" and not something we customise for a single view in crm.

Reply:
done

Reply:
Not the right "IMP" commit :D

The doc can be updated when you introduce and test the functionality, sorry if that was unclear.

Reply:
Sorry, I think that I'll need to change my glasses :/